pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/extension/PythonDetails.java

```diff
@@ -127,10 +126,10 @@ public String normalizePythonVersion(String pythonVersion) {
 
         /* Ensure that it's okay to use this version (major/minor only) of Python. */
         System.out.println("PV:" + pythonVersion
-                           + " WL:" + Arrays.asList(PythonVersion.whitelistedPythonVersions)
+                           + " WL:" + PythonVersion.whitelistedPythonVersions
                            + " M.N:" + new PythonVersion(pythonVersion).getPythonMajorMinor());
 
```

Comment:
I assume the println's are for debugging only? They need to be removed before a merge. In general if we want to print something, we need to use a Logger so that it can be at levels based on the type of info.